applications/concept-cat/src/main/java/no/ccat/dto/ConceptAllMessage.java

```diff
@@ -0,0 +1,16 @@
+package no.ccat.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import org.springframework.util.MultiValueMap;
+
+@EqualsAndHashCode(callSuper = false)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Data
+public class ConceptAllMessage extends HarvestTriggerMessage {
```

Comment:
Consider a more descriptive and to-the-point name

Reply:
I'm gonna run it by Stig. These names are from the spec

Reply:
If you need me to change the specification, give me a hint.